ui/server/services/EmailService/email.service.ts

```diff
@@ -68,7 +68,7 @@ export default {
 
     const roundLink = appLink(`/${group.slug}/${round.slug}`);
 
-    const adminEmails: SendEmailInput[] = admins.map((admin) => ({
+    const adminEmails: SendEmailInput[] = admins.map((admin) => admin.user?.emailSettings?.roundJoinRequest ? ({
```

Comment:
I have a memory that we should do it like `?? true` like in the other cases to catch the case where a user doesn't have `emailSettings` yet

Reply:
I think user always have email settings. If a user didn't set email settings, postgres will return the default settings. I have tested a case, where user didnt have email settings. 

By default roundJoinRequest is true

Reply:
In the user model it's defined as `emailSettings    EmailSettings?`, so it shouldn't exist for a new user. When fetching it to render the settings page it's upserted so that might be why it worked for you, but I suspect it might crash for a user that's never visited the settings page.